### DISCUSSION TITLE
Add the ability to send Admin Commands to a channel in Discord

### DISCUSSION THREAD

**Discussion by elusive-ttv** [gameserverapp/Platform#498] [Suggestions]
I would love to have the ability to send admin command usage to a channel in discord, it would be a big help to monitor our team!

As a tag along to this suggestion, being able to send sales notifications to discord would also be a HUGE bonus!

**Comment by zerafox** [comment]
Sending in-game admin commands to discord would not only help monitoring your Admin team but can also provide proof to a playerbase there is no cheating going on by Admins either since a record is printed for them as well as server owners to review.

**Comment by mvaessen** [comment]
i forgot to update this one, but this has been added a while back. Thanks for the suggestion
